### ISSUE TITLE
getAll methods will crash after executed update notification on ios

### ISSUE THREAD

**Issue by mrichie** [katzer/cordova-plugin-local-notifications#473]
test step: 
1 、cordova.plugins.notification.local.schedule(options);
2 、cordova.plugins.notification.local.update(options);
3 、cordova.plugins.notification.local.getAll(function(obj){ console.log(obj); })
this will crash my app!
```
2015-03-19 16:57:39.324 Baby[40724:2494422] (
        (
                {
            at = 1426755404;
            badge = 8424;
            data = "{\"notifyID\":153,\"msgID\":8735,\"type\":\"alarm\"}";
            id = 153;
            sound = "file://alarm.wav";
            text = "\U5c3f\U6e7f\U5566";
            title = "\U5c3f\U6e7f\U63d0\U9192";
            updatedAt = "2015-03-19 08:57:14 +0000";
        }
    )
)
2015-03-19 16:57:39.325 Baby[40724:2494422] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Invalid type in JSON write (__NSDate)
```
I'm using the latest plugin version, and tested on ios8.1.3

**Comment by sergeh** [comment]
@mrichie I seem to be having the same issue as you but I get it right after I update a notification (see #496)
Have you found a solution?

**Comment by katzer** [comment]
Fixed: 0157a9f4507857777640a8a41fbc1eedca744b3e
